jupyter/d2l-java/test_notebook.sh

```diff
@@ -21,3 +21,8 @@ if [[ $# -eq 0 ]]; then
 else
     run_test $1
 fi
+
+--ExecutePreprocessor.timeout= 600
```

Comment:
This doesn't work.
```
jupyter nbconvert --to html --execute --ExecutePreprocessor.timeout=600 --output $base $1
```
This should work. Please try to run this line against your notebook locally first to see if you can resolve issue

Reply:
You can refer to this issue: 
https://github.com/jupyter/nbconvert/issues/256
To have more resolution on how you can manipulate on time, perhaps -1 will bring you unlimited timeout

Reply:
Yeah I was making that change only. Checked out nbconvert documentation.